src/main/java/millom/sandbox/pojos/Descriptions.java

```diff
@@ -0,0 +1,292 @@
+package millom.sandbox.pojos;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Descriptions {
```

Comment:
As all fields are set then using a `record` would have been better and way shorter that using a `class`.  
But no need to change anything as this works just fine!

Reply:
As discussed with @kmmatwork, for now left the class as same. From next time will try to make use of `record`.